src/lib/protocols/syslog.c

```diff
@@ -107,7 +108,7 @@ void init_syslog_dissector(struct ndpi_detection_module_struct *ndpi_struct, u_i
   ndpi_set_bitmask_protocol_detection("Syslog", ndpi_struct, detection_bitmask, *id,
 				      NDPI_PROTOCOL_SYSLOG,
 				      ndpi_search_syslog,
-				      NDPI_SELECTION_BITMASK_PROTOCOL_V4_V6_TCP_OR_UDP_WITH_PAYLOAD_WITHOUT_RETRANSMISSION,
+				      NDPI_SELECTION_BITMASK_PROTOCOL_V4_V6_UDP_WITH_PAYLOAD,
```

Comment:
Could you add a comment here about why TCP is missing, please? Just as a reminder....